server/lib/src/constants/schema.rs

```diff
@@ -731,7 +752,34 @@ pub static ref SCHEMA_CLASS_PERSON_DL5: SchemaClass = SchemaClass {
     systemmust: vec![
         Attribute::IdVerificationEcKey.into()
     ],
-    systemexcludes: vec![EntryClass::ServiceAccount.into()],
+    systemexcludes: vec![EntryClass::ServiceAccount.into(), EntryClass::Application.into()],
+    ..Default::default()
+};
+
+pub static ref SCHEMA_CLASS_PERSON_DL6: SchemaClass = SchemaClass {
```

Comment:
Minor, but just an artefact of the review taking a while is these need to target DL7 now, not DL6 https://github.com/kanidm/kanidm/blob/master/server/lib/src/server/migrations.rs#L654

Reply:
Updated.